tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -47,11 +49,88 @@ reset_req(size_t data_len)
 	alloc_req(data_len);
 }
 
+static void
+alloc_resp(size_t data_len)
+{
+	BUG_ON(resp);
+	resp = test_resp_alloc(data_len);
+}
+
+static void
+free_resp(void)
+{
+	if (resp)
+		test_resp_free(resp);
+	resp = NULL;
+}
+
+static void
+reset_resp(size_t data_len)
+{
+	free_resp();
+	alloc_resp(data_len);
+}
+
+static void
+free_msgs(void)
+{
+	free_req();
+	free_resp();
+}
```

Comment:
Some of the functions above are used only once in the code and actually just a 2-line wrappers - why do you use them? Too many and too similar (e.g. alloc_resp() and test_resp_alloc()) makes problems to understand the code - a code reader must keep them all in their head.